test/powershell/Modules/Microsoft.WSMan.Management/ConfigProvider.Tests.ps1

```diff
@@ -0,0 +1,436 @@
+Describe "WSMan Config Provider" -Tag Feature,RequireAdminOnWindows {
+    BeforeAll {
+        #skip all tests on non-windows platform
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        $PSDefaultParameterValues["it:skip"] = !$IsWindows
+
+        if ($IsWindows) {
+            $badCredentialError = 1326
+            $pluginXml = [xml](winrm g winrm/config/plugin?name=microsoft.powershell -format:xml)
+            $pluginPath = "WSMan:\localhost\Plugin\microsoft.powershell"
+            $testPluginXml = [xml]($pluginXml.OuterXml)
+            $testPluginXml.PlugInConfiguration.Name = "TestPlugin"
+            $testPluginXml.PlugInConfiguration.RemoveAttribute("xml:lang")
+            $testPluginXml.PlugInConfiguration.Resources.Resource.ResourceUri = "http://schemas.microsoft.com/powershell/TestPlugin"
+            $testPluginXml.PlugInConfiguration.Resources.Resource.Security.Uri = "http://schemas.microsoft.com/powershell/TestPlugin"
+            Set-Content $TestDrive\plugin.xml -Value $testPluginXml.OuterXml
+            $null = winrm c winrm/config/plugin?Name=TestPlugin -file:$TestDrive\plugin.xml
+            $testPluginPath = "WSMan:\localhost\Plugin\TestPlugin"
+            $testUser = (Get-Random)
+            $testPass = "Secret123!"
+            $null = net user $testUser $testPass /add
+        }
+    }
+
+    AfterAll {
+        $Global:PSDefaultParameterValues = $originalDefaultParameterValues
+        if ($IsWindows) {
+            $null = winrm d winrm/config/plugin?Name=TestPlugin
+            $null = net user $testUser /DELETE
```

Comment:
Maybe use as in line 50 `> $null`?

Reply:
I'll change it to `$null =` for consistency